### DISCUSSION TITLE
Support for NeoForge (1.21) in issues?

### DISCUSSION THREAD

**Discussion by ghost** [RakambdaOrg/FallingTree#704] [General]
Was about to post a bug report (in a similar vein to RakambdaOrg/FallingTree#689 and RakambdaOrg/FallingTree#685) but I see there's only a Fabric template.

So, where should I report a crash relating specifically to Falling Tree trying to load client-only classes on a server? :D Would like to avoid annoying the dev(s) by filling in junk the the required `Fabric API version` field of the fabric-bug-report template...

Cheers!

**Comment by Rakambda** [comment]
You can report it as forge for now, just mention it's neoforge. I'll fix the templates later.

**Reply by Rakambda** [reply]
If forge is buggy too then use the fabric template as best as you can 😅

**Comment by ghost** [comment]
Will do, thanks much! :beer:
